packages/microsoft_sqlserver/manifest.yml

```
@@ -0,0 +1,33 @@
+format_version: 1.0.0
+name: microsoft_sqlserver
+title: "Microsoft SQL Server"
+version: 0.1.0
+license: basic
+description: "Package to integrate Microsoft SQL Server audit events."
```

Collect audit events from Microsoft SQS Server with Elastic Agent.
```suggestion
description: Collect audit events from Microsoft SQL Server with Elastic Agent.
```